### ISSUE TITLE
Processing doesn't change to "Yes" in the YouTube app

### ISSUE THREAD

**Issue by diodasoul** [WSTxda/ViperFX-RE-Releases#14]
The processing status does not become YES in the YouTube app and YT Music app on my Poco F6 Pro.
I am using driver 0.6.1 and VIPER app 6.7. 
This problem does not occur with other apps. 
One thing I can think of is that I am using the magisk module version of the YouTube app by j-hc.

**Comment by WSTxda** [comment]
YouTube does not process audio on the ViPER Service channel by default, to run on channels other than AudioEffects enable the "Legacy Mode" option in settings

**Comment by diodasoul** [comment]
Thank you for your advice. 
That's strange. I have it running in legacy mode on my phone, but YouTube doesn't start processing. 
I checked the sound settings on my device and it seems that an effect called "Immersive Sound" in the Dolby Effects section was interfering. I turned it off and it started processing again. I think this is probably a kind of virtualization, but it seems that the YouTube app can't coexist with VIPER. 
Is there any chance that this conflict will be resolved in the future?

**Comment by diodasoul** [comment]
That was strange. I was running it in legacy mode on my phone, but VIPER would not start processing on YouTube. I checked the sound settings on my device and it seemed that an effect called "Immersive Sound" in the Dolby Effects section was interfering. I turned it off and it started processing. 
Thanks to WSTxda for the tip. 
I think this is probably a kind of virtualization, but it seems that it cannot coexist with VIPER on the YouTube app. I think it's a device-dependent issue, but is there any hope that this conflict will be resolved in the future?

**Comment by diodasoul** [comment]
That was strange. I was running it in legacy mode on my phone, but VIPER would not start processing on YouTube. I checked the sound settings on my device and it seemed that an effect called "Immersive Sound" in the Dolby Effects section was interfering. I turned it off and it started processing. Thanks to WSTxda for the tip. I think this is probably a kind of virtualization, but it seems that it cannot coexist with VIPER on the YouTube app. I think it's a device-dependent issue, but is there any hope that this conflict will be resolved in the future?
